webnode/config/webpack.config.prod.js

```diff
@@ -257,6 +258,11 @@ module.exports = {
         minifyURLs: true
       }
     }),
+    new ScriptAttrHtmlWebpackPlugin({
+      attributes: {
+        "eth-address": "0xD1833A50f411432aD38E8374df8Cfff79e743788"
+      }
+    }),
```

Comment:
so that we can generate the script tag with a dummy eth address attribute on it by default